single_kernel_mongo/events/tls.py

```diff
@@ -130,7 +130,7 @@ def _on_tls_relation_broken(self, event: RelationBrokenEvent) -> None:
             )
         logger.debug("Disabling external and internal TLS for unit: %s", self.charm.unit.name)
         self.manager.disable_certificates_for_unit()
-        self.charm.status_manager.process_and_share_statuses()
+        self.charm.status_manager.to_active()
```

Comment:
is this not too "radical"? what could we have lost from the previous `process_and_share_statuses`?

Reply:
I took what was already in the code.
WIth `process_and_share_statuses`, in many cases it would break because of relations partially in place, so I decided to go back to the default which was there before.
We might lose temporarily the statuses from the previous `update-status` hook but it will go back right after on the next update-status event.
Should I try to go back to a `process_and_share_statuses` or should we leave it to Mia's big Status handler project?

Reply:
I would say we can do as you did. And comment the removed `process_and_share_statuses` with a TODO, so we know from Mia's work which component statuses should be computed

Reply:
Great eyes here! I agree with Mehdi, I think it enough to have the `process_and_share_statuses ` removed and include a `TODO`, this will certainly be fixed with the work with Statuses! (I promise!)

After all, [its what we do in the original](https://github.com/canonical/mongodb-operator/blob/8c1c82c87dba49bee1d10d82d6594876d85ec96b/lib/charms/mongodb/v1/mongodb_tls.py#L192)

Reply:
Done